fast64_internal/oot/data/oot_getters.py

```diff
@@ -0,0 +1,37 @@
+from xml.etree.ElementTree import parse as parseXML, Element
+
+
+def getRoot(xmlPath: str) -> Element:
+    """Parse an XML file and return its root element"""
+    try:
+        return parseXML(xmlPath).getroot()
+    except:
+        from ...utility import PluginError
+
+        raise PluginError(f"ERROR: File '{xmlPath}' is missing or malformed.")
+
+
+def getEnumList(dataList: list, customName: str):
```

Comment:
it's weird that this expects dataList elements to have id, key, name

I'd either embrace supporting two different classes that happen to have id, key, name defined
```py
from .oot_actor_data import OoT_Actor
from .oot_object_data import OoT_ObjectData
def getEnumList(dataList: list[OoT_Actor | OoT_ObjectData.OoT_Object], customName: str):
```
this may have circular import issues, that could be fixable with e.g. `from . import oot_actor_data` and annotate with `"oot_actor_data.OoT_Actor"` (notice it's a string)

or create a base dataclass which OoT_Actor and OoT_Object derive from (a better idea imo)
```py
@dataclass
class OoT_DataElement: # name???
    id: str
    key: str
    name: str
def getEnumList(dataList: list[OoT_DataElement], customName: str):
```
```py
@dataclass
class OoT_Object(OoT_DataElement):
    pass # nothing else to define :shrug: 
```